src/screens/Exchange/Exchange.js

```diff
@@ -979,6 +995,32 @@ class ExchangeScreen extends React.Component<Props, State> {
     paymentNetworkBalance: null,
   }));
 
+  generateBTCAssetOption = () => {
+    const symbol = 'BTC';
+    const {
+      btcAddresses,
+      btcBalances,
+      baseFiatCurrency,
+      rates,
+    } = this.props;
+    const { address } = btcAddresses[0];
+    const addressBalance = btcBalances[address];
+    const rawAssetBalance = addressBalance ? satoshisToBtc(addressBalance.confirmed) : 0;
+    const assetBalance = rawAssetBalance ? formatAmount(rawAssetBalance) : null;
+    const formattedBalanceInFiat = getFormattedBalanceInFiat(baseFiatCurrency, assetBalance, rates, symbol);
+    const btcAsset = initialAssets.find(e => e.symbol === symbol);
+    return {
+      key: symbol,
+      value: symbol,
+      icon: btcAsset ? btcAsset.iconUrl : '',
+      iconUrl: btcAsset ? btcAsset.iconUrl : '',
```

Comment:
`btcAsset` is always present in `initialAssets` so we can just deconstruct `iconUrl` in above and avoid code duplicate.

Reply:
flow acted up on that case since find can return undefined